### ISSUE TITLE
Use modifier key on its own

### ISSUE THREAD

**Issue by throwaway1037** [swaywm/sway#6058]
### Please read the following before submitting:
- We are not accepting any new window management features unless they get implemented by i3.

### Please fill out the following:
- **Description:**
Please describe in plain English what the enhancement is and what the use case is.

Is it possible to use the modifier key on its own, for instance: binding the super key on its own to open up Rofi?

**Comment by emersion** [comment]
Please ask questions on IRC.
